cosmwasm/packages/wormhole-bindings/src/query.rs

```diff
@@ -0,0 +1,42 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::{Binary, CustomQuery, Empty};
+
+#[cw_serde]
+pub struct Signature {
+    /// The index of the guardian in the guardian set.
+    pub index: u16,
```

Comment:
the sdk defines the `Signature` type with a `uint8` -- https://github.com/wormhole-foundation/wormhole/blob/dev.v2/sdk/vaa/structs.go#L77-L82

Any reason to use `u16` here instead of `u8` for the guardian index?